test/Project.toml

```
@@ -59,4 +61,4 @@ StatsFuns = "0.9.5, 1"
 TimerOutputs = "0.5"
 Tracker = "0.2.11"
 Zygote = "0.5.4, 0.6"
-julia = "1.3"
+julia = "1.6"
```

This should be
```suggestion
julia = "1.7"
```
but probably it's even better to remove it completely
```suggestion
```
to ensure that Project.toml and test/Project.toml don't diverge again.